update docs to describe usage of COLORS env variable (#1148)
testing-library/testing-library-docs: docs/dom-testing-library/api-debugging.mdx

you'd like a solution that works for both, see
[`cross-env`](https://www.npmjs.com/package/cross-env).

> **Note**: The output of the DOM are colorized by default if you tests are
> running in a node environment. However, you may sometimes wish to turn off
> colors, such as in cases where the output is being written to a log file for
> debugging purposes. You can use the environment variable `COLORS` to explicitly
> force the colorization off or on. For example:

Reviewer: ```suggestion
**Note**: The output of the DOM is colorized by default if your tests are
running in a node environment. However, you may sometimes want to turn off
colors, such as in cases where the output is written to a log file for
debugging purposes. You can use the environment variable `COLORS` to explicitly
force the colorization off or on. For example:
```

```
COLORS=false npm test
```

> As with `DEBUG_PRINT_LIMIT`, `COLORS` works in macOS/Linux but you'll need to do
> something else for Windows. If you'd like a solution that works for both, see
> [`cross-env`](https://www.npmjs.com/package/cross-env).

Reviewer: Sorry for the additional comment.
I think we can just copy paste this note, what do you think?

```suggestion
This works on macOS/Linux, you'll need to do something else for Windows. If
you'd like a solution that works for both, see
[`cross-env`](https://www.npmjs.com/package/cross-env).
```

Author: Yeah, sure.
